cre.cpp

```diff
@@ -3511,29 +3518,19 @@ static int getImageDataFromPosition(lua_State *L) {
 }
 
 static int setUserHyphenationDict(lua_State *L) {
-    CreDocument *doc = (CreDocument*) luaL_checkudata(L, 1, "credocument");
```

Comment:
If these really don't require a document, and if you remove this, they should be part of `cre_func` like the other `cre.*Hyph*` stuff (that means, a function and not a method of the document) - and you should call them as `cre.setUserHyphenationDict(...)`. (As they would be used in a reader*.lua module loaded when a cre document is loaded, you don't need any `require("libkoreader-cre")` as `cre` has already been loaded and made global.